website/docs/md_components/divider_components/usage_markdown.md

```diff
@@ -20,17 +20,17 @@ Too many lines can overload the screen with visual noise and create unnecessary
 
 ### When to use 
 
-- When whitespace is not enough of a separator. 
+- when whitespace is not enough of a separator. 
```

Comment:
Why does this listing item have a period when the next items do not end in a period?
Again keep it consistent everywhere